tests/karate/src/test/java/be/utils/JsonConverter.java

```diff
@@ -0,0 +1,85 @@
+package be.utils;
+
+
+import com.intuit.karate.Json;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Base64;
+
+public class JsonConverter {
+
+  public Json fromMapToJson(Map<String,String> map){
+    return Json.of(map);
+  }
+
+  private String senderPk = "J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM=";
+  private String signature = "ONylxgHA9cbsB_lwdfbn3iyzRd4aTpJhBMnvEKhmJF_niE_pUHdmjxDXjEwFyvo5WiH1NZXWyXG27SYEpkasCA==";
+  private String messageIdForced = "";
+
+  public String convertJson(Json json){
+
+    String stringJson = json.toString();
+    byte[] jsonBytes = stringJson.getBytes(StandardCharsets.UTF_8);
+    Base64.Encoder encoder = Base64.getEncoder();
+    String base64 = encoder.encodeToString(jsonBytes);
+    return base64;
+  }
+
+
```

Comment:
I think some documentation would help understand which part of the protocol this generates